mkdocs.yml

```diff
@@ -21,6 +21,7 @@ nav:
 
 # Configuration
 theme:
+  favicon: img/favicon.ico
```

Comment:
add `logo: img/logo.svg` above/below this line

Reply:
Close #86  added in extra logo.svg file and changes to mkdoc.yml